encoding/mvt/prepare.go

```diff
@@ -73,26 +57,26 @@ func ScaleGeo(geo tegola.Geometry, tile *tegola.Tile) geom.Geometry {
 	return nil
 }
 
-func scalept(g geom.Point, tile *tegola.Tile) geom.Point {
+func preparept(g geom.Point, tile *tegola.Tile) geom.Point {
 	pt, err := tile.ToPixel(tegola.WebMercator, g)
 	if err != nil {
 		panic(err)
 	}
 	return geom.Point(pt)
 }
 
-func scalelinestr(g geom.LineString, tile *tegola.Tile) (ls geom.LineString) {
+func preparelinestr(g geom.LineString, tile *tegola.Tile) (ls geom.LineString) {
 	pts := g
 	// If the linestring
 	if len(pts) < 2 {
 		// Not enought points to make a line.
 		return nil
 	}
 	ls = make(geom.LineString, 0, len(pts))
-	ls = append(ls, scalept(pts[0], tile))
+	ls = append(ls, preparept(pts[0], tile))
 	lidx := len(ls) - 1
 	for i := 1; i < len(pts); i++ {
-		npt := scalept(pts[i], tile)
+		npt := preparept(pts[i], tile)
 		if cmp.PointEqual(ls[lidx], npt) {
```

Comment:
@ear7h let's purge this piece. The `prepare` process should not add or remove any points.